home-page.html

```diff
@@ -5,17 +5,18 @@
         <link rel="stylesheet" href="css/home-page.css">
         <title>ontarget - home</title>
         <link rel="icon" href="images/favicon.ico">
+        <script defer src="product-cards.js"></script>
     </head>
-
     <!--[1] TODO: Adds links once they are ready -->
+    <!--[2] FIXME: change all the spans (span) to links (a) -->
+    <!--[3] TODO: still need to write the something in the footer -->
```

Comment:
You must ALWAYS place comments as near as possible to the referenced part.
You should only use the "[<n>]" method only if many places around the code need the same comment.

In this case you should either make the todos more generic and use it for both sections or just place them as before

Reply:
i think i fixed it

Reply:
> ou should only use the "[n]" method only if many places around the code need the same comment.

This means that if the comment is related to only one line you don't need the number.

Also the 2 comments almost say the same thing... "TODO: Temporary text, remember to change it once it will be available"

Lo cambio io

Reply:
Same applies for the other 2 comments.
Both must remember you to make them clickable and to add links but you decided to be more specific so that's ok.

Reply:
Another info, there is no "better" way between duplicating comments or using the "[n]" method...
I just wanted you to try both then you'll decide what to use once you get more experience